files/source/galaxy_workflows.yml

```diff
@@ -0,0 +1,15 @@
+workflows:
+  - trs_id: "https://dockstore.org/api/ga4gh/trs/v2/tools/#workflow/github.com/iwc-workflows/rnaseq-pe/main/versions/v0.9"
+    type: "REGULATION"
+    description: "Workflows for the analysis of ChIP-seq, ATAC-Seq, and beyond."
```

Comment:
are these descriptions for the workflows? or for the 'type' ?

there will eventually be many workflows belonging to a 'type' (or whatever we end up calling it.. type is too vague imo), and i wonder if this source of duplication implies a need for a new yaml file?

Reply:
Yes, reading the descriptions, I agree they should like they describe the workflow type rather than a specific workflow within the type. I think this came from having only one workflow per type for the initial kickoff. Sounds like we do need a type (or "modality?) tab.

Reply:
I think we could also add (or generate) links to view the workflow details in GitHub, Dockstore etc. or wherever they live.

Reply:
Can/should we align these 'types' with the collection as organized on dockstore (and iwc site)?  (https://dockstore.org/organizations/iwc)

This can be programmatically fetched if so.